web-app/js/smartR/_angular/directives/fetchButton.js

```diff
@@ -8,6 +8,8 @@ window.smartRApp.directive('fetchButton',
         return {
             restrict: 'E',
             scope: {
+                loaded: '=',
```

Comment:
This is far more safe than binding it to the button state, which is what happens right now.